aspnetcore-webapi/RemoteHostIpAddress/RemoteHostIpAddress/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,80 @@
+using Microsoft.AspNetCore.Mvc;
+using System.Net.Sockets;
+using System.Net;
+
+namespace RemoteHostIpAddress.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
+    {
+        private static readonly string[] Summaries = new[]
+        {
+          "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+        };
+
+        private readonly ILogger<WeatherForecastController> _logger;
+
+        public WeatherForecastController(ILogger<WeatherForecastController> logger)
+        {
+            _logger = logger;
+        }
+
+        [HttpGet(Name = "GetWeatherForecastData")]
+        public IEnumerable<WeatherForecast> Get()
+        {
+            var remoteIpAddress = GetRemoteHostIpAddressUsingRemoteIpAddress();
+            Console.WriteLine(remoteIpAddress);
+
+            return Enumerable.Range(1, 5).Select(index => new WeatherForecast
+            {
+                Date = DateOnly.FromDateTime(DateTime.Now.AddDays(index)),
+                TemperatureC = Random.Shared.Next(-20, 55),
+                Summary = Summaries[Random.Shared.Next(Summaries.Length)]
+            });
+        }
+
+        public IPAddress GetRemoteHostIpAddressUsingRemoteIpAddress()
+        {
+            return HttpContext.Connection.RemoteIpAddress;
+        }
+
+        public IPAddress? GetRemoteHostIpAddressUsingXForwardedFor()
+        {
+            IPAddress? remoteIpAddress = null;
+            var forwardedFor = HttpContext.Request.Headers["X-Forwarded-For"].FirstOrDefault();
+
+            if (!string.IsNullOrEmpty(forwardedFor))
+            {
+                var ips = forwardedFor.Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries)
+                                      .Select(s => s.Trim());
```

Comment:
Can't we use this version of the Split https://learn.microsoft.com/en-us/dotnet/api/system.string.split?view=net-7.0#system-string-split(system-char-system-stringsplitoptions)
Since it take one characater, without needing the table : `new char[] { ',' }`

Reply:
sure, thanks :)